proto/observer/params.proto

```diff
@@ -11,6 +11,7 @@ message CoreParamsList {
   repeated CoreParams core_params = 1;
```

Comment:
Can we remove this struct all together ?

Reply:
What do you mean? Is it related to your comment above?

Reply:
Yes , resolving this